### ISSUE TITLE
[Feature Request] A floating button mode

### ISSUE THREAD

**Issue by goddade** [markusfisch/PieLauncher#99]
Is it possible to add a new floating button mode? 
Add a button that stays on the screen to open the pie menu. Pie menu can even be run without being a launcher. 
This can make pie launcher more open and suitable for more usage scenarios.
For example, 
  solve #98 , #83, #12 through other apps, 
  solve #52, #36, #9 through multiple buttons.
I haven't seen any similar floating button app on Android.
Thanks for your awesome work.

**Comment by markusfisch** [comment]
Yes, this is very much possible! And I'm already playing with this idea… 😉
